conda/gateways/connection/adapters/jlap/no_cache.py

```diff
@@ -1,4 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
```

Comment:
2012?

Reply:
I have wondered why we list 2012 as well; my understanding is that we're just sticking with listing the year Anaconda got founded vs when any particular module or feature was written/modified.

Reply:
There is a bit of content out there that describes the practice, e.g. [here](https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code) or [here](https://opensource.google/documentation/reference/copyright#the_year). Which more or less agree on that when a file is created it should get the current year to denote the start of the automatic copyright and then not changed when edited, moved etc.

That said, I'm not a lawyer, people in legal are though, so let's ask them!

Reply:
not bothered by it, copyrights expire too far in the future for 10 years to matter